src/MiNET/MiNET/Player.cs

```diff
@@ -2238,7 +2238,11 @@ protected virtual void HandleItemUseTransactions(Transaction transaction)
 				case McpeInventoryTransaction.ItemUseAction.Use:
 					_itemUseTimer = Level.TickTime;
 					itemInHand.UseItem(Level, this, transaction.Position);
-					//Inventory.UpdateInventorySlot(transaction.Slot, transaction.Item);
+					var item = transaction.Item;
+					if (item is ArmorItem)
+					{
```

Comment:
I think you can fast equip not only armor but also elytra and other stuff

Reply:
Also, doesn't client just send normal transaction from inventory 0 to armor slots when tapping? I always thought it works exactly like that o.o

Reply:
Indeed. I tried to fast-equip pumpkins, heads & elytra, apparently, only armor and elytra can be fast-equipped.